README.md

```diff
@@ -96,6 +96,14 @@ After that you can login to DataLens on http://localhost:8080 using the default
 
 You can use the same credentials to configure Zitadel and add new users using Zitadel control panel at http://localhost:8085/. **Don't forget to login there at least once to change the default password.**
 
+By default in DataLens with authentication every new user has datalens.viewer role which allows viewing collections and workbooks in read only mode. You are not allowed to modify or create any new entity with this role. In order to be able to edit or create new collection or workbook you need to have datalens.editor or datalens.admin role. To grant roles please open Zitadel at http://localhost:8085/ui/console/grants, then find user to whome you want to grant a new role, click on user and select new role.
+
+DataLens supports the following roles:
+
+datalens.viewer - allows view collections and workbooks of any user and doesn't allow to edit or create new ones.
+datalens.editor – includes datalens.viewer role and allows to edit, delete and create new collection or workbook for any user. 
```

Comment:
`for any user` -> `in any place`?

Reply:
discussed offline, decided leave as it is